docs-2.0/3.ngql-guide/11.edge-statements/4.delete-edge.md

```diff
@@ -0,0 +1,24 @@
+# DELETE EDGE
+
+```ngql
+DELETE EDGE <edge_type> <src_vid> -> <dst_vid>[@<rank>] [, <edge_type> <src_vid> -> <dst_vid>[@<rank>] ...]
+```
+
+Use `DELETE EDGE` to delete edges. The `DELETE EDGE` statement deletes one edge or multiple edges at a time. You can use `DELETE EDGE` together with pipe. For more information about pipe, see [Pipe operator](../5.operators/4.pipe.md).
+
+## Examples
+
+```ngql
+nebula> DELETE EDGE serve "player100" -> "team200";
```

Comment:
it is the same as DELETE EDGE serve "player100" -> "team200" @ 0.

Reply:
rank is not supposed to be documented here